Authy2FA/App_Start/IdentityConfig.cs

```diff
@@ -55,10 +55,10 @@ public static ApplicationUserManager Create(IdentityFactoryOptions<ApplicationUs
             manager.PasswordValidator = new PasswordValidator
             {
                 RequiredLength = 6,
-                RequireNonLetterOrDigit = true,
+                RequireNonLetterOrDigit = false,

```

Comment:
Thanks for pointing that reducing the strength of the password is not valid because it _somehow_ easy the development process. I restored the default values for `PasswordValidator`, please have a look at 28422ce